desktop/ui/src/main/java/org/datacleaner/windows/ElasticSearchDatastoreDialog.java

```diff
@@ -82,7 +80,6 @@ public ElasticSearchDatastoreDialog(WindowContext windowContext, MutableDatastor
         setSaveButtonEnabled(false);
 
         _clientTypeComboBox = new JComboBox<>(ClientType.values());
-        _clientTypeComboBox.setSelectedItem(DEFAULT_CLIENT_TYPE);
```

Comment:
That's actually more of a failing in line 223, where I used ClientType.REST instead.